src/ai/backend/test/tester/README.md

```diff
@@ -3,3 +3,76 @@
 This package provides the test runner and result management logic for the testing framework. The tester is responsible for executing testcases, managing concurrency, and exporting results.
 
 The tester package is the orchestrator that brings together testcases, templates, and contexts to perform automated testing and collect results.
+
+Unlike other test codes, the Tester package does not mock any components required for the Backend.AI environment or interactions with those components at all.
+
+It performs actual integration tests by directly sending requests to the real Backend.AI manager and webserver.
+
+## How to run tester-based integration tests in CLI
+
+To run all the tests based on Tester, execute the following command.
+
+```console
+$ backend.ai test run-all
+```
```

Comment:
The most recommended method in the current local environment is `backend.ai test run`, so please make that part come first.